src/main/resources/lang/strings_en.properties

```diff
@@ -589,6 +589,9 @@ colors.mines=Mines
 colors.allies=Allies
 colors.enemies=Enemies
 colors.npcs=NPCs
+colors.low_relays=LoW Relays
+colors.space_balls=SpaceBalls
+colors.static_entities=Static Entities
```

Comment:
I think a total of 0 users are going to understand this category
maybe just call it "other entities" and use it as a generic fallback color?